lib/flat_search_web/live/page_live.ex

```diff
@@ -18,10 +24,14 @@ defmodule FlatSearchWeb.PageLive do
       |> Filters.change_filter(params)
       |> Map.put(:action, :insert)
 
-    params = Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end)
+      ic_params = Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end)
```

Comment:
ic_params?

Reply:
text there is formatted to be case insensitive, naming is hard :P ic -> ci?

Reply:
is there any need to keep this and params separately?

Reply:
Later on you can see use of both original params and the ic_params.

Reply:
I'll look into it and try to get rid of ic

Reply:
That's my another idea but not sure if that makes a huge difference :p